specification/flag-evaluation/flag-evaluation.md

```diff
@@ -151,21 +151,19 @@ FlagEvaluationDetails<MyStruct> myStructDetails = client.getObjectDetails<MyStru
 
 ##### Requirement 1.13
 
-> The `evaluation details` structure's `variant` field **SHOULD** contain a identifier logically corresponding to the return flag value.
-
-For example, the flag value might be `3.14159265359`, and the variant field's value might be `"pi"`.
+> In cases of normal execution, the `evaluation details` structure's `variant` field **MUST** contain the value of the `variant` field in the `flag resolution` structure returned by the configured `provider`.
 
 ##### Requirement 1.14
 
-> The `evaluation details` structure's `reason` field **SHOULD** indicate the semantic reason for the returned flag value. Possible values vary by provider, but may include `"TARGETING_MATCH"`, `"SPLIT"`, `"DISABLED"`, `"DEFAULT"`, `"UNKNOWN"` and `"ERROR"`.
+> In cases of normal execution, the `evaluation details` structure's `reason` field **MUST** contain the value of the `reason` field in the `flag resolution` structure returned by the configured `provider`.
 
 ##### Requirement 1.15
 
-> In cases of normal execution, the `evaluation details` structure's `error code` field **MUST** not be set, or otherwise must contain a null or falsy value.
+> In cases of normal execution, the `evaluation details` structure's `error code` field **MUST** contain the value of the `error code` field in the `flag resolution` structure returned by the configured `provider`.
```

Comment:
Ya, this one is a bit tricky - maybe it's an edge case. I was thinking of a scenario wherein the provider didn't throw or otherwise abnormally terminate, but set the error. Maybe we dont need to define such a situation.

Reply:
I will remove it entirely.